### DISCUSSION TITLE
v0.5.3

### DISCUSSION THREAD

**Discussion by dail8859** [dail8859/NotepadNext#163] [New Releases]
## What's Changed
* Add 'Find All in All Documents'
* Add 'Replace All in All Open Documents'
* Add ability to navigate to search results
* Add Swedish translation
* Add command line argument to load a specified translation
* Fix segfault when trying to open non-existing file
* Fix allowing user to create non-existing file
* Ensure lines are visible when going to range
* Fix crash when dragging text into the application
* Automate publishing GitHub release

**Full Changelog**: https://github.com/dail8859/NotepadNext/compare/v0.5.2...v0.5.3

<hr /><em>This discussion was created from the release <a href='https://github.com/dail8859/NotepadNext/releases/tag/v0.5.3'>v0.5.3</a>.</em>

**Comment by harvanek** [comment]
Hello, 

I would like to ask how to add my own language. In my case language for making TODO lists, I am using original win version and I already have user defined language in xml file, is it some how possible to use the xml file and import language to project. 
Thank you for your answer. 

Michal.

**Reply by dail8859** [reply]
Currently there's no support for user defined languages. At some point in the future I think having something like that would be very useful. Especially if it can import the xml definitions from Notepad±±

**Reply by harvanek** [reply]
Thank you for your fast answer, Yes, this feature would be amazing.
